src/evaluate.ts

```diff
@@ -436,7 +436,7 @@ async function loadFile(
     } catch (e) {
       outputWindow.append(`; Evaluation of file ${fileName} failed: ${e}`);
       if (res.stacktrace) {
-        outputWindow.saveStacktrace(res.stacktrace);
+        outputWindow.saveStacktrace(res.stacktrace.stacktrace);
```

Comment:
The first stacktrace is an object, which contains a stacktrace property, which refers to an array.